ldms/src/test/Makefile.am

```diff
@@ -39,4 +39,29 @@ test_metric_SOURCES = test_metric.c
 test_metric_LDADD = -lldms
 test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
-TESTS = $(check_PROGRAMS)
+EXTRA_DIST=ldms-run-static-tests.sh.in
+EXTRA_DIST+=ldms-static-test-list.sh.in
+EXTRA_DIST+=ldms-static-test-bypass.in
+
+CLEANFILES=ldms-run-static-tests.sh
+CLEANFILES+=ldms-static-test-list.sh
+CLEANFILES+=ldms-static-test-bypass 
+
+%.sh: %.sh.in
+	$(do_subst) < $< > $@
+	chmod 755 $@
+
+installcheck-local: ldms-run-static-tests.sh
```

Comment:
What are you trying to do here? Are you trying to exectute ldms-run-static-tests.sh?  If so, you need to put this on the next line. If you really meant for it to be a prerequisite of the installcheck-local target, then this should not be necessary because ldms-run-static-tests.sh should already have been generate at configure time by AC_CONFIG_FILES.

Reply:
Okay, I was mainly following the format from ldms/src/Makefile.am for the ovis-ldms-configvars.sh.in file. No, it doesn't need to be executed, its is supposed to be a prerequisite of installcheck-local.